frontend/src/metabase/list-view/components/ListView/ColumnValue.tsx

```diff
@@ -190,6 +190,10 @@ export function ColumnValue({
       );
     case isQuantity(column):
     case isScore(column): {
+      if (!column?.settings?.["show_mini_bar"]) {
+        return <Text fw="bold">{value}</Text>;
```

Comment:
Shouldn't we just pass through to line 303?
Or  render just `MiniBarCell` conditionally.